### DISCUSSION TITLE
Liaex SCORM 1.2 & 2004 Export - Empty page loading

### DISCUSSION THREAD

**Discussion by michaelkowalczyk** [LiaScript/LiaScript#194] [General]
Hi,
I sucessfully exported a SCORM 1.2 Package as described in https://www.npmjs.com/package/@liascript/exporter  
using "liaex -i project/README.md --format scorm1.2 --output rockOn"
I confirmed that the zip file contains all the liascript files.
I imported the zip into the university Ilias LMS as well as https://rusticisoftware.com/products/scorm-cloud/
The file was imported sucessfully.
When opening the course no content is shown, only a box to select and load a course. (see screenshot)

Same experience with the 2004 export.

Thanks for any advice
Michael

**Comment by michaelkowalczyk** [comment]
![Screenshot - 10_09_2024 , 15_28_12](https://github.com/user-attachments/assets/6ca8878d-eeea-4767-8d85-9e6c8a385590)

**Reply by SebastianZug** [reply]
Same problem with Firefox - it was necessary to delete all cached files. Afterward, the course overview worked fine again.

**Comment by michaelkowalczyk** [comment]
Update: using another prompt "liaex -i course/README.md -f scorm2004 --scorm-masteryScore 80 --scorm-iframe" the scorm loads in the lms as it should.
